lib/Minz/ModelPdo.php

```diff
@@ -96,10 +109,10 @@ public function __construct($currentUser = null, $currentPdo = null) {
 			try {
 				$this->dbConnect();
 				return;
-			} catch (PDOException $e) {
+			} catch (Minz_PDOConnectionException $e) {
```

Comment:
We also need to catch other PDO exceptions

Reply:
This case is tagged to "ok" by phpstan PDOException not

Reply:
Maybe, but that probably hides another problem, because this is wrong